libs/openFrameworksCompiled/project/ios/Debug.xcconfig

```diff
@@ -5,8 +5,9 @@ GCC_ENABLE_SSE3_EXTENSIONS = YES
 GCC_ENABLE_SUPPLEMENTAL_SSE3_INSTRUCTIONS = YES
 GCC_INLINES_ARE_PRIVATE_EXTERN = NO
 GCC_MODEL_TUNING = G5
+GCC_OPTIMIZATION_LEVEL = 0
```

Comment:
I wonder if this is causing the issue on older mac's to crash with the simulator (too much data?)